### PR TITLE
Update bridge-lpt address.md

### PR DESCRIPTION
**Pull request by NightWolf92**
Swap the Addresses to reflect the proper order for migrating on Bridge.Arbitrum.io

### CODE REVIEW

**Review by hthillman — CHANGES_REQUESTED** [review]
```
enter the L1 LPT contract address ([0x289ba1701C2F088cf0faf8B3705246331cB8A839]
```
can you change this to "L2 LPT contract address"

otherwise looks good!

**Comment by NightWolf92** [comment]
updated the wording to add L2 Arbitrum LPT Contract address.

Line 12 has the L2 Arbitrum address in there

Line 16 has the L1 LPT Address in there.

The address ending in ab239 was the one I needed to go from L1 to L2 so the wording seems to read properly.

File: docs/delegators/how-to-guides/bridge-lpt.md [hunk 1]
```diff
@@ -9,8 +9,8 @@ This guide will walk tokenholders through the process of moving LPT from the Eth
...
+2. Click "Token", and enter the L2 Arbitrum LPT contract address ([0x58b6a8a3302369daec383334672404ee733ab239](/protocol/reference/deployed)). If you are using testnet, use the `LivepeerToken` address for Arbitrum Rinkeby.
 3. Select LPT from the dropdown. Once you've done this, you should see your L1 balance.
 4. Click `Deposit` to move your L1 LPT to L2.
 
-If you need to move from L2 to L1, the same instructions apply; the only difference is that you should start with your wallet connected to Arbitrum (Arbitrum Rinkeby if using testnet) and enter the L1 LPT contract address ([0x58b6a8a3302369daec383334672404ee733ab239](/protocol/reference/deployed)) as the destination address.
\ No newline at end of file
+If you need to move from L2 to L1, the same instructions apply; the only difference is that you should start with your wallet connected to Arbitrum (Arbitrum Rinkeby if using testnet) and enter the L1 LPT contract address ([0x289ba1701C2F088cf0faf8B3705246331cB8A839](/protocol/reference/deployed)) as the destination address.
```
Comment (hthillman) [inline docs/delegators/how-to-guides/bridge-lpt.md:16]: this isn't the L1 contract address, it's 0x58

Comment (hthillman) [reply docs/delegators/how-to-guides/bridge-lpt.md:16]: so this should read "enter the L2 LPT contract address"

**Pull request merged**